### ISSUE TITLE
SupervisedWandbRunner doesn't support inference

### ISSUE THREAD

**Issue by dlazerka** [catalyst-team/catalyst#603]
**Describe the bug**
SupervisedWandbRunner requires a logdir, while `runner.predict_loader()` doesn't accept it.

```
  File "/home/dl/.pyenv/versions/ve/lib/python3.6/site-packages/catalyst/dl/runner/supervised.py", line 291, in predict_loader
    check=check
  File "/home/dl/.pyenv/versions/ve/lib/python3.6/site-packages/catalyst/dl/runner/supervised.py", line 249, in infer
    self.run_experiment(experiment, check=check)
  File "/home/dl/.pyenv/versions/ve/lib/python3.6/site-packages/catalyst/contrib/runner/wandb.py", line 140, in run_experiment
    self._pre_experiment_hook(experiment=experiment)
  File "/home/dl/.pyenv/versions/ve/lib/python3.6/site-packages/catalyst/contrib/runner/wandb.py", line 60, in _pre_experiment_hook
    monitoring_params["dir"] = str(Path(experiment.logdir).absolute())
  File "/home/dl/.pyenv/versions/3.6.8/lib/python3.6/pathlib.py", line 1001, in __new__
    self = cls._from_parts(args, init=False)
  File "/home/dl/.pyenv/versions/3.6.8/lib/python3.6/pathlib.py", line 656, in _from_parts
    drv, root, parts = self._parse_args(args)
  File "/home/dl/.pyenv/versions/3.6.8/lib/python3.6/pathlib.py", line 640, in _parse_args
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```
**To Reproduce**
```
runner.predict_loader(
        model=model,
        loader=loader,
        resume=config.checkpoint_path,
        verbose=True
    )
```

**Comment by CrafterKolyan** [comment]
Reproducible example:
```
from catalyst import dl
import torch
import torch.nn as nn
from torch.utils.data import DataLoader, TensorDataset

X = torch.rand(16, 10)
y = torch.rand(X.shape[0])
model = nn.Linear(X.shape[1], 1)
dataset = TensorDataset(X, y)
loader = DataLoader(dataset, batch_size=8)
runner = dl.SupervisedWandbRunner()

runner.predict_loader(
    model=model,
    loader=loader
)
```

Consider that:
```
from catalyst import dl
import torch
import torch.nn as nn
from torch.utils.data import DataLoader, TensorDataset

X = torch.rand(16, 10)
y = torch.rand(X.shape[0])
model = nn.Linear(X.shape[1], 1)
dataset = TensorDataset(X, y)
loader = DataLoader(dataset, batch_size=8)
runner = dl.SupervisedRunner()

runner.predict_loader(
    model=model,
    loader=loader
)
```
is working perfectly.

**Comment by CrafterKolyan** [comment]
By the way, I am not sure this is actually should be supported. It's really strange how you want your results to be shown in WandB so actually I think you should use `SupervisedRunner` instead of `SupervisedWandbRunner` for `predict_loader`

**Comment by Scitator** [comment]
fixed `20.02.3` :)

**Comment by CrafterKolyan** [comment]
@Scitator I don't see this one fixed in `20.02.4`. Also I don't see any commits which tried to fix this problem in `20.02.3`. My example from above (https://github.com/catalyst-team/catalyst/issues/603#issuecomment-582924740) is still not working. Error is the same. 

Please reopen this issue.

![image](https://user-images.githubusercontent.com/9883873/75760313-264f9680-5d48-11ea-8c58-d8d9e69d4e95.png)
